src/khoj/interface/web/config.html

```diff
@@ -224,40 +254,20 @@ <h3 class="card-title">
     var configure = document.getElementById("configure");
     configure.addEventListener("click", function(event) {
         event.preventDefault();
-        configure.disabled = true;
-        configure.innerHTML = "Configuring...";
-        const csrfToken = document.cookie.split('; ').find(row => row.startsWith('csrftoken'))?.split('=')[1];
-        fetch('/api/update?&client=web', {
-            method: 'GET',
-            headers: {
-                'Content-Type': 'application/json',
-                'X-CSRFToken': csrfToken
-            }
-        })
-        .then(response => response.json())
-        .then(data => {
-            console.log('Success:', data);
-            document.getElementById("status").innerHTML = "Configured successfully!";
-            document.getElementById("status").style.display = "block";
-            configure.disabled = false;
-            configure.innerHTML = "⚙️ Configured";
-        })
-        .catch((error) => {
-            console.error('Error:', error);
-            document.getElementById("status").innerHTML = "Unable to save configuration. Raise issue on Khoj Discord or Github.";
-            document.getElementById("status").style.display = "block";
-            configure.disabled = false;
-            configure.innerHTML = "⚙️ Configure";
-        });
+        updateIndex(false, "Configured successfully!", "Unable to configure. Raise issue on Khoj Discord or Github.", configure, "Configuring...");
     });
 
     var reinitialize = document.getElementById("reinitialize");
     reinitialize.addEventListener("click", function(event) {
         event.preventDefault();
-        reinitialize.disabled = true;
-        reinitialize.innerHTML = "Reinitializing...";
+        updateIndex(true, "Reinitialized successfully!", "Unable to reinitialize. Raise issue on Khoj Discord or Github.", reinitialize, "Reinitializing...");
```

Comment:
We should prefix the loading, success and error messages with the emojis. Maybe we can prefix these messages with emojis in the `updateIndex` function itself for consistency?